docs/configuration.md

```diff
@@ -261,6 +261,7 @@ placement_policy:
 |------------------|----------|---------------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `default`        | `string` | yes           | `REP 3`       | Default policy of placing containers in NeoFS. If a user sends a request `CreateBucket` and doesn't define policy for placing of a container in NeoFS, the S3 Gateway will put the container with default policy. |
 | `region_mapping` | `string` | yes           |               | Path to file that maps aws `LocationContraint` values to NeoFS placement policy. The similar to `--container-policy` flag in `neofs-s3-authmate` util, see in [docs](./authmate.md#containers-policy)             |
```

Comment:
`Constraint` here as well (separate fix).

Reply:
Extended

Reply:
Don't see this being fixed.

Reply:
I've added extra info inside authmade.md, now I think I was a mistake.
Could you please extend the message? What the `Constraint`? In this place already mentioned `LocationContraint`

Reply:
s/Contraint/Constraint/